client/src/components/pages/admin/Submitted_Id/index.js

```diff
@@ -1,5 +1,406 @@
+                          <Typography variant="body2" align="left">
+                            <Checkbox
+                              checked={data && data.freeCodeCampTopics}
+                              className="profile_check_box"
+                              color="primary"
+                              disabled
+                            />
+                            FreeCode Camp Topics
+                          </Typography>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td className="item" colSpan="2">
+                          <Typography variant="body2" align="left">
+                            <Checkbox
+                              checked={data && data.applicationSubmittedDate}
+                              className="profile_check_box"
+                              color="primary"
+                              disabled
+                            />
+                            Submitted
+                            {data.applicationSubmittedDate &&
+                              `on ${
+                                data.applicationSubmittedDate.split('T')[0]
+                              }`}
+                          </Typography>
+                        </td>
+                      </tr>
+                      <tr className="doted">
+                        <td className="item" colSpan="2">
+                          <Typography variant="body2" align="left">
+                            <Checkbox
+                              checked={acceptedVal}
+                              className="profile_check_box"
+                              color="primary"
+                              disabled
+                            />
+                            Accepted
+                          </Typography>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td colSpan="2" className="profile__btn">
+                          <div className="profile__btn__toggle">
+                            <Button onClick={() => history.goBack()}>
+                              Go Back
+                            </Button>
+                            <Button
+                              onClick={() => {
+                                setAccept(userId, acceptedVal);
+                                setAcceptedVal(!acceptedVal);
+                              }}
+                            >
+                              {acceptedVal ? 'Disapprove' : 'Accept'}
+                            </Button>
+                          </div>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                }
+              />
+            </div>
+          ) : (
+            <div className="loading">
+              <Limitation />
+            </div>
+          )}
+        </div>
+      </div>
+    </>
+  );
+};
+
```

Comment:
Important: 
The data you displayed is not enough to be seen by admin , 
try to display all data in database . 
I mean within Availability part you should display also 
university :
specialization: 
CV link :
Coding Experience: 
primary motivation: 
Motivation: 
CA Reaching :
--------------------- 
and please work on the style to be more acceptable .